.github/workflows/continuous-integration.yaml

```diff
@@ -107,6 +107,7 @@ jobs:
       - name: Integration Test
         env:
           COORD_IMAGE: 237073351946.dkr.ecr.us-east-1.amazonaws.com/stargateio/coordinator-dse-68
+          TEST_REGEX: '!VectorSearchIntegrationTest*'
```

Comment:
This is not being used, I think need to add to below:

     -Dit.test=$TEST_REGEX 

after `-DskipUnitTests` ?

Reply:
For testing I removed it temporarily

Reply:
Ah I should have looked at commits. It wouldn't have skipped the test etc.